modules/hub/src/ChannelsService.ts

```diff
@@ -106,10 +106,8 @@ export default class ChannelsService {
     }, user)
 
     if (hasPendingOps(channelStateStr)) {
-      LOG.info(
-        `User requested a deposit while state already has pending operations ` +
-        `(user: ${user}; current state: ${JSON.stringify(channelStateStr)})`
-      )
+      LOG.info(`User ${user} requested a deposit while state already has pending operations `)
+      LOG.debug(`[Request Deposit] Current state: ${JSON.stringify(channelStateStr)}`)
```

Comment:
I kept similar hub logs in `info` log-level minus the channel state. I didn't remove the channel state logging but switched it to only print on `debug` log-level. The default log-level is `info` so these won't be printed unless we explicitly start the hub in `debug` log-level-mode.